redis/marketplace_token_test.go

```diff
@@ -28,7 +29,8 @@ func setUpFakeToken() *marketplace.BearerToken {
 
 // TestGetTokenBadTenant tests that when given a bad or nonexistent tenant, an expected error is returned.
 func TestGetTokenBadTenant(t *testing.T) {
-	mr, err := miniredis.Run()
+	mr := miniredis.NewMiniRedis()
```

Comment:
So this is probably a weird request - but could we spin this up in the `TestMain` package that way it's there for all the tests? Or is there a reason we're spinning up a new one every time?

Reply:
Yes, that is a good idea. I initially tried to do that, but I faced some problems I thought they were caused by Miniredis being run at the package level. But it seems that it was something else. Implemented the start/stop at the package level. Thanks!